src/components/Footer/Footer.tsx

```diff
@@ -44,15 +44,15 @@ export const Footer: React.FC = () => {
         <nav className={styles.legal}>
           <a
             className={styles.legalAnchor}
-            href="imprint.html"
+            href={process.env.PUBLIC_URL + 'imprint.html'}
             target="_blank"
             rel="noreferrer"
           >
             Imprint
           </a>
           <a
             className={styles.legalAnchor}
-            href="terms.html"
+            href={process.env.PUBLIC_URL + '/terms.html'}
```

Comment:
Does the public url have a trailing slash or not? You've added the slash in front of terms but not imprint.

Reply:
It does. I was testing it and extra slash was left there by mistake. I fixed it in the next commit.